appyters/dimensionality_reduction_visualization/requirements.txt

```diff
@@ -7,8 +7,13 @@ pandas
 matplotlib
 sklearn
 seaborn
+scanpy
 IPython
 bokeh
 umap-learn
-MulticoreTSNE git+https://github.com/maayanlab/Multicore-TSNE
-react-scatter-board @ git+https://github.com/maayanlab/react-scatter-board@v2.1
\ No newline at end of file
+scipy
+qnorm
+multiprocess
```

Comment:
I don't see where this is used?

Reply:
Removed multiprocessing. qnorm was for normalization and scipy was for leiden algorithm.